project.php

```diff
@@ -492,13 +492,12 @@ function do_project_info_table()
 
     $formatted_now = strftime($time_format, time());
     $ct = _("Current Time");
-    $current_time_addition = "&nbsp;&nbsp;&nbsp;($ct: $formatted_now)";
+    $current_time_addition = "&nbsp;&nbsp;&nbsp;(" . html_safe($ct) . ": " . html_safe($formatted_now) . ")";
```

Comment:
I think this would be a bit easier to read if we just pulled out the spaces-for-formatting from the strings:
```php
$current_time_addition = "&nbsp;&nbsp;&nbsp;" . html_safe("($ct: $formatted_now)");
```

Reply:
Updated here and in sandbox. You're suggestion is much better :).